docker-compose.yaml

```diff
@@ -60,27 +60,21 @@ services:
       PORT_glpi: 3306
       ENGINE_glpi: "mysql@dbgate-plugin-mysql"
 
-  # To use as a LDAP connection inside GLPI, you'll need to set the following values:
-  # server: openldap
-  # port: 1389
-  # connection filter: (objectClass=inetOrgPerson)
-  # base dn: dc=example,dc=org
-  # Use bind: yes
-  # Root DN: cn=admin,dc=example,dc=org
-  # Password: adminpassword
-  # Login field: uid
-  # Synchronization field: entryuuid
-  # TODO: create a bin/console dev command that create a LDAP server object with
-  # the information above to make things even easier ;)
+  # To create a preconfigured LDAP object in GLPI, run make console c='tools:generate_dev_ldap'
   openldap:
-    image: bitnami/openldap:latest
+    image: osixia/openldap:latest
+    restart: "unless-stopped"
     environment:
-      - LDAP_ADMIN_USERNAME=admin
-      - LDAP_ADMIN_PASSWORD=adminpassword
-      - LDAP_USERS=user01,user02
-      - LDAP_PASSWORDS=password1,password2
-      - LDAP_ROOT=dc=example,dc=org
-
+      LDAP_ORGANISATION: GLPI
+      LDAP_DOMAIN: glpi.org
+      LDAP_ADMIN_PASSWORD: admin
```

Comment:
Have you try to remove these in order to be able to use the same ldif as in tests ?

Reply:
I can't because it will use the default values instead, which is example.org I think.

If we want to reuse the same file, maybe we could split the original ldif in two files (Tree root + LDAP administrator in a file, the rest in another)?

This way the CI include both files and the container only the second one.